adblock-lean

```diff
@@ -495,16 +505,16 @@ restart_dnsmasq()
 
 export_existing_blocklist()
 {
-	if [[ -f /tmp/dnsmasq.d/.blocklist.gz ]]
+	if [[ -f /var/run/adblock-lean/dnsmasq.d/.blocklist.gz ]]
```

Comment:
does it need to be in subdirectory dnsmasq.d? or is this just an oversight?

Reply:
Oops! Good catch. Will fix.